packages/components/src/styles/copyright/copyright.css

```diff
@@ -6,4 +6,14 @@
     content: var(--copyright);
     text-shadow: 0 1px 3px #515151bf;
   }
+
+  &--variant {
```

Comment:
I suggest removing `&--variant`. 
`sd-copyright--vertical` is already very descriptive

Reply:
Avoid making changes in this area. This syntax is also used in other components, such as the container. If the decision is to remove the &--variant, we should create a new ticket to address that.

Reply:
We only have variants for color things. We have booleans on other examples as well (see e. g. the headline, hidden-links and interactive). Therefore I second @smfonseca .

Reply:
for map-marker, we also have variants (cluster, main, place). And that was only a quick look in one of the components 🤔
So i would recommend: 
1. New ticket for removing variant for the whole library if its not color related.

Reply:
But there are more than two options, and therefore some kind of "prefix" is needed. It could be called "shape" for "map-marker", which feels kinda artifical.

But here only `horizontal` and `vertical` can exist. And then we usually take booleans.

Reply:
If you really want to stick to a prefix, at least it should not be variant, but `orientation` in this case:

![CleanShot 2024-09-20 at 06 49 05@2x](https://github.com/user-attachments/assets/5982f189-9f4a-4dde-a1dd-820aa3a7b324)

Would be fine with that approach.